File: backends/fiber/internal/models/measurement.go
package models

import "github.com/google/uuid"

type Measurement struct {
	ID     uuid.UUID       `gorm:"column:measurement_id;primaryKey;default:gen_random_uuid()" json:"id"`
	Name   string          `gorm:"column:name;not null" json:"name"`
	TypeID uuid.UUID       `gorm:"column:type_id;not null" json:"-"`
	Type   MeasurementType `json:"type"`
	System string          `gorm:"column:system;not null" json:"system"`
}

File: backends/aspnet/Recipes.API/entities/Measurement.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Recipes.API.entities;

public partial class Measurement
{
    [JsonPropertyName("id")]
    public Guid MeasurementId { get; set; }

    public string Name { get; set; } = null!;

    public Guid TypeId { get; set; }

    public string System { get; set; } = null!;

    public virtual ICollection<Ingredient> Ingredients { get; set; } = new List<Ingredient>();

    public virtual ICollection<StepIngredient> StepIngredients { get; set; } = new List<StepIngredient>();

    public virtual MeasurementType Type { get; set; } = null!;
}
